tests/auth_tests/test_auth_backends.py

```diff
@@ -803,6 +814,126 @@ def test_clean_credentials_sensitive_variables(self):
+        )
+
+        # Check Client.post function
+        self.assertContains(
+            response,
+            '<tr><td>data</td><td class="code">'
+            "<pre>&#x27;********************&#x27;</pre></td></tr>",
+            html=True,
+            status_code=500,
+        )
+
+        # Check RequestFactory.post function
+        self.assertContains(
+            response,
+            '<tr><td>post_data</td><td class="code">'
+            "<pre>&#x27;********************&#x27;</pre></td></tr>",
+            html=True,
+            status_code=500,
+        )
+
+    def test_setpasswordform_validate_passwords_sensitive_variables(self):
+        try:
+            password_form = SetPasswordForm(AnonymousUser())
+            password_form.cleaned_data = {
+                "password1": "test_password1",
+                "password2": "test_password2",
+            }
+            password_form.validate_passwords()
+        except ValueError:
+            exc_info = sys.exc_info()
+
+        rf = RequestFactory()
+        response = technical_500_response(rf.get("/"), *exc_info)
+
+        self.assertContains(
+            response,
+            '<tr><td>password1</td><td class="code">'
+            "<pre>&#x27;********************&#x27;</pre></td></tr>",
+            html=True,
+            status_code=500,
+        )
+
+        self.assertContains(
+            response,
+            '<tr><td>password2</td><td class="code">'
+            "<pre>&#x27;********************&#x27;</pre></td></tr>",
+            html=True,
+            status_code=500,
+        )
+
+    @override_settings(
+        AUTH_PASSWORD_VALIDATORS=[
+            {"NAME": __name__ + ".TypeErrorValidator"},
+        ]
+    )
+    def test_setpasswordform_validate_password_for_user_sensitive_variables(self):
+        try:
+            password_form = SetPasswordForm(AnonymousUser())
+            password_form.cleaned_data = {"password2": "test_password2"}
+            password_form.validate_password_for_user(AnonymousUser())
+        except TypeError:
+            exc_info = sys.exc_info()
+
+        rf = RequestFactory()
+        response = technical_500_response(rf.get("/"), *exc_info)
+
+        self.assertContains(
+            response,
+            '<tr><td>password</td><td class="code">'
+            "<pre>&#x27;********************&#x27;</pre></td></tr>",
+            html=True,
+            status_code=500,
+        )
+
+    def test_passwordchangeform_clean_old_password_sensitive_variables(self):
+        try:
+            password_form = PasswordChangeForm(User())
+            password_form.cleaned_data = {"old_password": "test_old_password"}
+            password_form.error_messages = None  # Done to help raise a TypeError
+            password_form.clean_old_password()
+        except TypeError:
+            exc_info = sys.exc_info()
+
+        rf = RequestFactory()
+        response = technical_500_response(rf.get("/"), *exc_info)
```

Comment:
```suggestion
        response = technical_500_response(rf.get("/"), *exc_info)
        self.assertNotContains(response, "test_old_password", status_code=500)
```
This makes this test fail also